tests/Exercise/BlorkemonCardsTest.elm

```diff
@@ -0,0 +1,229 @@
+    max card1.power card2.power
+
+sortByMonsterName : List Card -> List Card
+sortByMonsterName =
+    List.sortBy .monster
+
+sortByCoolness : List Card -> List Card
+sortByCoolness =
+    List.sortWith
+        (\\c1 c2 ->
+            case (c1.shiny, c2.shiny) of
+                (True, False) -> LT
+                (False, True) -> GT
+                _ -> compare c2.power c1.power
+        )
+
+compareShinyPower : Card -> Card -> Order
+compareShinyPower card1 card2 =
+    case compare card1.power card2.power of
+        EQ ->
+            case (card1.shiny, card2.shiny) of
+                (True, False) -> GT
+                (False, True) -> LT
+                _ -> EQ
+        byPower -> byPower
+
+expectedWinner : Card -> Card -> String
+expectedWinner card1 card2 =
+    case compareShinyPower card1 card2 of
+        GT -> card1.monster
+        LT -> card2.monster
+        EQ -> "too close to call"
+"""
+        ]
+
+
+noMax : Test
+noMax =
+    test "maxPower doesn't use max" <|
+        \() ->
+            """
+module BlorkemonCards exposing (..)
+
+maxPower card1 card2 =
+    if isMorePowerful card1 card2
+    then card1.power
+    else card2.power
+"""
+                |> Review.Test.run BlorkemonCards.maxPowerUsesMax
+                |> Review.Test.expectErrors
+                    [ TestHelper.createExpectedErrorUnder (Comment "maxPower doesn't use max" "elm.blorkemon-cards.use_max" Essential Dict.empty) "maxPower"
+                    ]
+
+
+noSortBy : Test
+noSortBy =
+    test "sortByMonsterName doesn't use List.sortBy" <|
+        \() ->
+            """
+module BlorkemonCards exposing (..)
+
+sortByMonsterName =
+    List.sortWith
+        (\\c1 c2 ->
+            if c1.name < c2.name then
+                LT
+            else if  c1.name == c2.name then
+                EQ
+            else
+                GT
+        )
+"""
+                |> Review.Test.run BlorkemonCards.sortByMonsterNameUsesSortBy
+                |> Review.Test.expectErrors
+                    [ TestHelper.createExpectedErrorUnder (Comment "sortByMonsterName doesn't use List.sortBy" "elm.blorkemon-cards.use_sort_by" Essential Dict.empty) "sortByMonsterName"
+                    ]
+
+
+noCompareShinyPower : Test
+noCompareShinyPower =
+    test "expectedWinner doesn't use compareShinyPower" <|
+        \() ->
+            """
+module BlorkemonCards exposing (..)
+
+expectedWinner card1 card2 =
+    case compare ( card1.power, shinyValue card1.shiny ) ( card2.power, shinyValue card2.shiny ) of
+        EQ ->
+            "too close to call"
+
+        LT ->
+            card2.monster
+
+        GT ->
+            card1.monster
+"""
+                |> Review.Test.run BlorkemonCards.expectedWinnerUsesCompareShinyPower
+                |> Review.Test.expectErrors
+                    [ TestHelper.createExpectedErrorUnder (Comment "expectedWinner doesn't use compareShinyPower" "elm.blorkemon-cards.use_shiny_power" Essential Dict.empty) "expectedWinner"
```

Comment:
This looks better than the way I did it, as it doesn't rely on the position of the error in the string. I'll have to try and do it like this next time ...